src/main/java/uk/ac/ebi/eva/commons/models/data/VariantAnnotation.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.data;
+
+import org.opencb.biodata.models.variant.annotation.ConsequenceType;
+import org.opencb.biodata.models.variant.annotation.Xref;
+import org.opencb.biodata.models.variation.PopulationFrequency;
+
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Slim version of {@link org.opencb.biodata.models.variant.annotation.VariantAnnotation}
+ * Unused fields removed.
+ *
+ */
+public class VariantAnnotation {
+
+    private String chromosome;
+
+    private int start;
+
+    private int end;
+
+    private String referenceAllele;
+
+    private String alternativeAllele;
+
+    private String id;
+
+    private List<Xref> xrefs;
+
+    private List<String> hgvs;
+
+    private List<ConsequenceType> consequenceTypes;
+
+    private List<PopulationFrequency> populationFrequencies = null;
```

Comment:
The populations it calculates frequencies for are 1000 Genomes and ExAC, which we have archived at EVA and calculated ourselves. So the ones from VEP would be redundant for us.